app/components/works/version_description_component.rb

```diff
@@ -13,8 +13,12 @@ def work_version
       form.object.model.fetch(:work_version)
     end
 
-    def render?
+    def show?
       %w[deposited rejected version_draft].include? work_version.state
     end
+
+    def show_hidden?
```

Comment:
I'm finding this method name confusing. (Is the show section hidden? Or are we showing something hidden?) I can't figure out what a good name would be however, something like `new_version_not_applicable?` or `user_version_na?` or `use_fallback_user_version?`